autoPlay.user.js

```diff
@@ -253,6 +260,10 @@ function disableParticles() {
 }
 
 function MainLoop() {
+	cHours = cTime.getUTCHours();
+	cMins = cTime.getUTCMinutes();
+	timeLeft = 60 - cMins;
```

Comment:
Since you made all those vars global, make a function to update and retrive them.

function updateCurrentTime() {
    cHours = cTime.getUTCHours();
    cMins = cTime.getUTCMinutes();
    timeLeft = 60 - cMins;
}
function isNearEndGame() {
    if (cHours == 15 && timeLeft <= minsLeft) {
        return true;
    }
    else {
        return false;
    }
}